README.adoc

```diff
@@ -11,41 +11,11 @@ See the link:https://docs.servicetalk.io/[ServiceTalk docs] for more information
 
 == Getting Started
 
-Currently, all ServiceTalk released artifacts are available in a Bintray repo:
-https://dl.bintray.com/servicetalk/servicetalk/, but will be published to Maven Central
-link:https://github.com/apple/servicetalk/issues/845[soon].
+All ServiceTalk released artifacts are available in link:https://repo1.maven.org/maven2/io/servicetalk/[the Maven
```

Comment:
I would suggest reverting docs changes until this is actually true. We can update README.md after the release and after we test that everything resolves correctly with our sample projects.